plugin-gradle/src/test/java/com/diffplug/gradle/spotless/TestFixtures.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2016 DiffPlug
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.diffplug.gradle.spotless;
+
+import java.util.Locale;
+
+import com.diffplug.common.base.StandardSystemProperty;
+
+interface TestFixtures {
+	boolean IS_WIN = StandardSystemProperty.OS_NAME.value().toLowerCase(Locale.US).contains("win");
+	String EXPECTED_RUN_SPOTLESS_APPLY_SUGGESTION = IS_WIN
+			? "Run 'gradlew.bat :spotlessApply' to fix these violations."
+			: "Run './gradlew :spotlessApply' to fix these violations.";
```

Comment:
We build and run tests on Java 8 and 11, I think this will fail on Java 8.  We just switched CI providers, looks it wasn't weren't configured properly for PRs.